README.md

```diff
@@ -255,3 +255,68 @@ See the [metrics documentation](metrics).
 
 ## Kata Admission controller webhook
 See the [webhook documentation](kata-webhook).
+
+## Using Vagrant to test your code changes
+
+It is strongly recommended that you test your changes locally before opening
+the pull request as this can save people's time and CI resources. Because
+testing Kata Containers involve complex build and setup instructions, scripts
+on the .ci directory are created to easy and provide a reproducible process; but they
+are meant to run on CI environments that can be discarded after use. Therefore, some
+developers may find it dangerous to run those scripts on their workstations.
+
+That said, we provide on this repository a Vagrantfile which allows developers to use
+the [vagrant](https://www.vagrantup.com) tool to create a VM with the setup as close as
+as possible to the environments where CI jobs will run the tests. Thus, allowing to
+reproduce a CI job locally.
+
+Currently it is only able to create a Fedora 32 VM. And your workstation must be capable
+of running VMs with:
+ * 8GB of system memory
+ * ~45 GB of disk space for the vm images on the Libvirt's storage pool
```

Comment:
- This section isn't very clear to me. Are you saying the user needs to have "8GB of spare memory for VM(s)"?
- s/vm/VM/
- This mentions "images", but how many images are required?
- Is there a missing bullet about 4 CPUs?